prime-router/src/main/kotlin/azure/observability/event/AzureEventUtils.kt

```diff
@@ -21,4 +21,17 @@ object AzureEventUtils {
             ObservationSummary(conditions)
         }
     }
+
+    /**
+     * For tracking the Message ID, we need the bundle.identifier value and system
+     * In FHIR bundle.identifier is optional so one or both may not be present
+     */
+    data class MessageID(val value: String? = null, val system: String? = null)
+
+    /**
+     * Returns the bundle identifier elements that relate to MSH-10 for tracking
```

Comment:
Should we mention MSH-10 here at all? Are we going to update this comment when we add support for other formats? My guess is no.

Reply:
Modified to `Returns the bundle identifier elements that relate to HL7v2 Message Header information for tracking` let me know if this is not general enough. (After pushing it I'm now thinking maybe some HL7 reports get this from something entirely different than message header, no idea).